src/wombats/components/configuration.clj

```diff
@@ -43,12 +44,16 @@
   []
   (let [file-location-dev (str (System/getProperty "user.dir") "/config/config.edn")
         file-location-prod (str (System/getProperty "user.home") "/.wombats/config.edn")]
-    (when (.exists (io/as-file file-location-dev))
-      (println "Using config at /config/config.edn")
-      file-location-dev)
-    (when (.exists (io/as-file file-location-prod))
-      (println "Using config at ~/.wombats/config.edn")
-      file-location-prod)))
+    (cond
```

Comment:
Order should be swapped here as well so the `prod` location has higher priority.